.github/workflows/pull-request-check-publish.yml

```diff
@@ -58,30 +58,38 @@ jobs:
 
       - name: Load Docker images
         run: |
-          docker load -i che-code.tgz
+          docker load -i che-code-amd64.tgz
+          docker load -i che-code-arm64.tgz
           docker load -i che-dev.tgz
 
       - name: Login to Quay.io
-        uses: docker/login-action@v3
+        uses: docker/login-action@v2
         with:
           registry: quay.io
           username: ${{ secrets.QUAY_PULL_REQUESTS_USERNAME }}
           password: ${{ secrets.QUAY_PULL_REQUESTS_PASSWORD }}
 
-      - name: Push che-code docker image
+      - name: Push che-code-amd64 docker image
         run: |
-          export IMAGE=quay.io/che-incubator-pull-requests/che-code:pr-${{env._PR_NUMBER}}-amd64
-          docker tag che-code ${IMAGE}
+          export IMAGE=quay.io/che-incubator/che-code:pr-${{env._PR_NUMBER}}-amd64
```

Comment:
if I understand correctly - after these changes main tags of the che-code are going to be mixed with pull requests tags - is it expected change?

Reply:
Maybe I misunderstand something, but PR images have `pr` prefix in its tag

Reply:
ah, I see. I will fix that

Reply:
thank you!
also please take a look at https://github.com/che-incubator/che-code/pull/496/files#diff-e32d596dd84d3d9ba0c7dfc413e06cca5172121444ac6438991aac1e3b116867R88
there is similar change